test-suite/hestonmodel.cpp

```diff
@@ -1440,7 +1440,7 @@ void HestonModelTest::testExpansionOnAlanLewisReference() {
 }
 
 void HestonModelTest::testExpansionOnFordeReference() {
-    BOOST_TEST_MESSAGE("Testing expansion on Forde reference prices...");
+    BOOST_TEST_MESSAGE("Testing expansion on Alan Lewis reference prices...");
```

Comment:
Hi Luigi,

no, I think the boot message was changed inadvertently and should remain as before.

Johannes